java/runtime/src/main/java/io/ray/runtime/RayDevRuntime.java

```diff
@@ -90,16 +90,16 @@ public void setAsyncContext(Object asyncContext) {
   @Override
   public PlacementGroup getPlacementGroup(
       PlacementGroupId id) {
-    //@TODO: We need a LocalGcsClient before implements this.
+    //@TODO(clay4444): We need a LocalGcsClient before implements this.
     throw new UnsupportedOperationException(
-      "Ray local running mode not support get Placement Group operation now.");
+      "Ray doesn't support placement group operations in local mode.");
   }
 
   @Override
-  public List<PlacementGroup> getAllPlacementGroup() {
-    //@TODO: We need a LocalGcsClient before implements this.
+  public List<PlacementGroup> getAllPlacementGroups() {
+    //@TODO(clay4444): We need a LocalGcsClient before implements this.
     throw new UnsupportedOperationException(
-      "Ray local running mode not support get all Placement Groups operation now.");
+      "Ray local running mode not support get all placement groups operation now.");
```

Comment:
Please update this line as well.

Reply:
It has updated, are you looking at a pre-version view?

Reply:
You added two `UnsupportedOperationException`s in this file and you've updated the message of only one of them.

Reply:
Ah, get it, I actually think you are talking about the Capitalization issues....